documentation/modules/exploit/windows/browser/exodus.md

```diff
@@ -0,0 +1,45 @@
+
```

Comment:
Please add a description of the vuln, where to download it, etc up here.

Reply:
Done.

Reply:
Hi h00die all the changes requested were made, is there anything else missing?